File: go/001/main.go
/*
 * ProjectEuler 001
 * Go / main.go
 * copyright (c) 2015 Susisu
 *
 * Go 1.4.1
 * `go build -o bin/main main.go`
 */


package main

import "fmt"

func main() {
    sum := 0
    for n := 1; n < 1000; n++ {
        if n % 3 == 0 || n % 5 == 0 {
            sum += n
        }
    }
    fmt.Printf("%d\n", sum)
}

File: cpp/001/main.cpp
/*
 * ProjectEuler 001
 * C++ / main.cpp
 * copyright (c) 2015 Susisu
 *
 * GCC / Apple LLVM version 6.0 (clang-600.0.56) (based on LLVM 3.5svn)
 * `g++ main.cpp -o bin/main`
 */

#include <iostream>

int main() {
    int sum = 0;
    for (int n = 1; n < 1000; ++n) {
        if (n % 3 == 0 || n % 5 == 0) {
            sum += n;
        }
    }
    std::cout << sum << std::endl;
    return 0;
}
